PlotTools/rake/meta.rake

```diff
@@ -82,7 +82,7 @@ namespace :meta do
     else
       # In MC, we can get the effective lumi from xsec and #events.
       file sample + '.lumicalc.sum' => sample + '.meta.json' do |t|
-        #sh "get_mc_lumi.py --sqrts #{sqrts} #{sample} `cat #{t.prerequisites} | extract_json.py n_evts` > #{t.name}"
+        sh "get_mc_lumi.py --sqrts #{sqrts} #{sample} `cat #{t.prerequisites} | extract_json.py n_evts` > #{t.name}"
```

Comment:
If you want to use it lets just un-deprecate it :). That's not a problem. 
(Can we for instance do a datadefs that is only used by LFV? That would rectify the issue that we had last time around with the datadef mainteinance)